CONTRIBUTING.rst

```diff
@@ -23,7 +23,7 @@ Running the Server
 Docker runs the server automatically. If you are not using Docker for 
 development, you can run it like this:
 
-- ``yarn serve``: starts development server (with watcher) at
+- ``make start``: starts development server (with watcher) at
   `<http://localhost:8080/>`_ (assets are served from ``dist/`` dir)
```

Comment:
```suggestion
  `<http://localhost:5000/>`_ (assets are served from ``dist/`` dir)
```

Reply:
Also I believe assets are no longer present in a `dist` dir. They should be coming from `static/` dirs located in the root dir or each individual app dir for the Django project.